bin/emqx

```diff
@@ -20,6 +20,9 @@ mkdir -p "$RUNNER_LOG_DIR"
 # Make sure data directory exists
 mkdir -p "$RUNNER_DATA_DIR"
 
+# cuttlefish try to read environment variables starting by "EMQX_, if not specified"
+export CUTTLEFISH_ENV_OVERRIDE_PREFIX="${EMQX_ENV_OVERRIDE_PREFIX:-EMQX_}"
```

Comment:
I myself was confused when reviewing the changes in .github.
Maybe we should use `CUTTLEFISH_ENV_OVERRIDE_PREFIX` everywhere instead of a mix.

```suggestion
export CUTTLEFISH_ENV_OVERRIDE_PREFIX="${CUTTLEFISH_ENV_OVERRIDE_PREFIX:-EMQX_}"
```